dbld/builddeps

```diff
@@ -11,10 +11,16 @@ set -e
 set -x
 
 function workaround_rpm_repos() {
+    MIRROR_URL='https://ftp.halifax.rwth-aachen.de/fedora/linux/'
```

Comment:
I'd probably remove the trailing slash as it will be duplicated after the below sed commands.

Reply:
I've removed the trailing slash.